src/app/components/Layout/Header/NavLinks/LinkItem.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+// styles
+import { Link, LinkItemWrapper } from './components';
+
+// types
+import INavLinkData from '../types/INavLinkData';
+
+type ILinkItemProps = INavLinkData;
+
+const LinkItem: React.FunctionComponent<ILinkItemProps> = ({ label, path }) => (
```

Comment:
Can't `INavLinkData` be used here? Also it can be shorter - `React.FC<INavLinkData>`